presentation/src/main/java/com/example/healthysalad/SelectBaseFragment.kt

```diff
@@ -33,8 +35,27 @@ class NavHostFragment : Fragment() {
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
+        //Data().loadItem("base")
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_nav_host, container, false)
+        var v : View = inflater.inflate(R.layout.fragment_select_base,container, false)
+
+        val openBackFrag: Button = v.findViewById(R.id.back_button)
+        openBackFrag.setOnClickListener {
```

Comment:
We can improve this code. Since this code is being used in two places, we can move it to a function and pass in the fragment instance as a parameter.

`
activity!!.supportFragmentManager
                 .beginTransaction().replace(R.id.activity_main, TitleFragment())
                 .addToBackStack(null).commit()
`

Reason: it improves code reusability, and prevents duplicates. Also, if we need to change it, we only need to change it in one place.